networking/enable-cluster-wide-proxy.adoc

```diff
@@ -8,22 +8,56 @@ toc::[]
 
 Production environments can deny direct access to the internet and instead have an HTTP or HTTPS proxy available. You can configure {product-title} to use a proxy by xref:../networking/enable-cluster-wide-proxy.adoc#nw-proxy-configure-object_config-cluster-wide-proxy[modifying the Proxy object for existing clusters] or by configuring the proxy settings in the `install-config.yaml` file for new clusters.
 
-== Prerequisites
+After you enable a cluster-wide egress proxy for your cluster on a supported platform, {op-system-first} populates the `status.noProxy` parameter with the values of the `networking.machineNetwork[].cidr`, `networking.clusterNetwork[].cidr`, and `networking.serviceNetwork[]` fields from your `install-config.yaml` file that exists on the supported platform. 
 
-* Review the xref:../installing/install_config/configuring-firewall.adoc#configuring-firewall[sites that your cluster requires access to] and determine whether any of them must bypass the proxy. By default, all cluster system egress traffic is proxied, including calls to the cloud provider API for the cloud that hosts your cluster. System-wide proxy affects system components only, not user workloads. Add sites to the Proxy object's `spec.noProxy` field to bypass the proxy if necessary.
-+
 [NOTE]
 ====
-The Proxy object `status.noProxy` field is populated with the values of the `networking.machineNetwork[].cidr`, `networking.clusterNetwork[].cidr`, and `networking.serviceNetwork[]` fields from your installation configuration with most installation types.
-
-For installations on Amazon Web Services (AWS), Google Cloud Platform (GCP), Microsoft Azure, and {rh-openstack-first}, the `Proxy` object `status.noProxy` field is also populated with the instance metadata endpoint (`169.254.169.254`).
+As a postinstallation task, you can change the `networking.clusterNetwork[].cidr` value but not the `networking.machineNetwork[].cidr` and the `networking.serviceNetwork[]` values. For more information, see "Configuring the cluster network range".
 ====
-+
+
+For installations on {aws-first}, {gcp-first}, {azure-first}, and {rh-openstack-first}, the `status.noProxy` parameter is also populated with the instance metadata endpoint, `169.254.169.254`.
+
+.Example of values added to the `status:` segment of a Proxy object by RHCOS
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: Proxy
+metadata:
+  name: cluster
+# ...
+networking:
+  clusterNetwork: <2>
+  - cidr: <ip_address_from_cidr>
+    hostPrefix: 23
+  network type: OVNKubernetes
+  machineNetwork: <3>
+  - cidr: <ip_address_from_cidr>
+  serviceNetwork: <4>
+  - 172.30.0.0/16
+# ...
+status:
+  noProxy:
+  - localhost
+  - .cluster.local
+  - .svc
+  - 127.0.0.1
+  - <api_server_internal_url> <1>
+# ...
+----
+<1> You can find the URL of the internal API server by running the `oc get infrastructures.config.openshift.io cluster -o jsonpath='{.status.etcdDiscoveryDomain}'` command. 
```

Comment:
This is something that's listed in the status section (I don't believe it's something that users should be able to modify). Is there a reason that they need to know how they can find something that they don't need to input?

Reply:
My 2 cents: it's always interesting to find where (and how) things come out, especially from a customer's perspective. This is to avoid customers' pain to look into the code, eventually. Many docs on the net lack this often because it's hard to track everything out.